index.js

```diff
@@ -339,3 +339,65 @@ module.exports.random = {
     return s * Math.pow(2, (-64 - e))
   }
 }
+
+/**
+ * Utils for simple signing and verification of HTTPS requests
+ **/
+module.exports.https = {
+  /**
+   * Sign the `message` with the given `secretKey`.
+   *
+   * @method
+   * @param {any} headers - headers containing the properties to sign
```

Comment:
i'm not sure if `any` is a valid jsdoc annotation. in any case it seems this should be `Object` https://jsdoc.app/tags-param